client/app/components/Modal/__tests__/Modal.spec.jsx

```diff
@@ -107,6 +123,84 @@ describe('Modal', () => {
         expect(wrapper.find('.modal').exists()).toEqual(false);
       });
     });
+
+    describe('closes when escape key is pressed', () => {
+      const component = (
+        <Modal
+          element="Hello"
+          body={bodyText}
+          title={title}
+          openListener={openListener}
+          onKeyPress={handleKeyPress}
+        />
+      );
+      it('toggles correctly', () => {
+        const wrapper = shallow(component);
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
+        expect(wrapper.find('.modal').exists()).toEqual(false);
+        wrapper.find('.modalElement').simulate('click');
+        expect(window.alert).toHaveBeenCalled();
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
+        expect(wrapper.find('.modal').exists()).toEqual(true);
+        wrapper.find('.modalBackdrop').simulate('keydown', { key: 'Escape' });
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
+        expect(wrapper.find('.modal').exists()).toEqual(false);
+      });
+    });
+
+    describe('closes when back drop is clicked', () => {
+      const component = (
+        <Modal
+          element="Hello"
+          body={bodyText}
+          title={title}
+          openListener={openListener}
+          onMouseLeave={handleMouseLeave}
+          onClick={handleOnClick}
+        />
+      );
+      it('toggles correctly', () => {
+        const wrapper = shallow(component);
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
+        expect(wrapper.find('.modal').exists()).toEqual(false);
+        wrapper.find('.modalElement').simulate('click');
+        expect(window.alert).toHaveBeenCalled();
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
+        expect(wrapper.find('.modal').exists()).toEqual(true);
+        wrapper.find('.modal').simulate('mouseleave');
+        wrapper.find('.modalBackdrop').simulate('click');
+        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
+        expect(wrapper.find('.modal').exists()).toEqual(false);
+      });
+    });
+
+    describe('closes when back drop is clicked', () => {
```

Comment:
Should this be `'does not close when back drop is not clicked'` instead?

Reply:
Yep, fixed. Good catch.